app/views/petitions/_rejected_petition.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>
+  <span class="heading-secondary">Rejected petition</span>
+  <%= @petition.title %>
```

Comment:
Thanks @alexander-bobin and @h-lame! Couldn't agree more, fixed here:  https://github.com/alphagov/e-petitions/commit/943535ae3adbc57ce531958fcac954c733c70f60